Foundation/NSGeometry.swift

```diff
@@ -33,7 +33,11 @@ public struct CGFloat {
     public var native: NativeType
     
     private var hash: Int {
+#if arch(i386) || arch(arm)
```

Comment:
I have my tree where I have done endian([big|little]) and intsize(nn) but reading the discussion I decided to postpone the submission. :)